<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellAccountItem</class>
 <widget class="QWidget" name="CellAccountItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="usernameLabel">
       <property name="styleSheet">
        <string notr="true">font-size:12pt; 
font-weight:700;
color: rgb(110, 112, 112);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clientIDLabel">
       <property name="styleSheet">
        <string notr="true">font-size:12pt; 
color: rgb(110, 112, 112);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="hostLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12pt; 
color: rgb(110, 112, 112);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font-size:12pt; 
color: rgb(110, 112, 112);</string>
           </property>
           <property name="text">
            <string> : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="styleSheet">
            <string notr="true">font-size:12pt; 
color: rgb(110, 112, 112);
</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="closeButton">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-style: outset;
	border-width: 0px;
	background-color: rgba(255,255,255,0);
}
QPushButton:pressed {
	background-color: rgba(255,255,255,0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/resources/resources/ic_delete_with_background.png</normaloff>:/resources/resources/ic_delete_with_background.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
